test/main.cpp

```diff
@@ -4,11 +4,83 @@
 #include "rs/flowcontrol/RSAnalysisEngineManager.h"
 #include "rs/flowcontrol/RSAnalysisEngine.h"
 
+#include <mongo/client/dbclient.h>
 
+#include "main.h"
+
+
+  uima::AnalysisEngine *engine;
 
 int main(int argc, char **argv)
 {
+  char *userEnv = getenv("USER");
+  std::string analysisEnginesArg, savePath;
+  std::vector<std::string> analysisEngines;
+  std::vector<std::string> analysisEngineFiles;
+  std::ostringstream engineList;
+  std::string engine_file;
+  
+  uima::ErrorInfo errorInfo;
+   mongo::client::GlobalInstance instance;
+  if(userEnv!=NULL)
+  {
+    ros::init(argc, argv, std::string("RoboSherlock_") +std::string(userEnv));
+
+  }
+  else
+  {
+    ros::init(argc, argv, std::string("RoboSherlock"));
+
+  }
+  ros::NodeHandle priv_nh;
+  priv_nh = ros::NodeHandle("~");
+
+  //ros::NodeHandle priv_nh = ros::NodeHandle("~");
+
+  priv_nh.param("ae", analysisEnginesArg, std::string("u_test"));
+  priv_nh.param("analysis_engines", analysisEnginesArg, analysisEnginesArg);
+  
+  analysisEngines.push_back(analysisEnginesArg);
```

Comment:
this whole section can be removed; We don't have multple engines; rs::common::getAEPaths(analysisEnginesArg, engine_file);